src/Cards/index.js

```diff
@@ -0,0 +1,76 @@
+import React, { useState, useEffect } from "react";
+import { CardsContainer, Score } from "./styled";
+
+import CurrentCard from "./CurrentCard";
+import Finish from "./Finish";
+import { triviaQuestions } from "./triviaQuestions";
+
+const total_questions = triviaQuestions.length;
+const currentRoundQuestions = [];
+
+const Cards = () => {
+  const [score, setScore] = useState(0);
+  const [currentQuestion, setCurrentQuestion] = useState(undefined);
+  const [isSubmit, setIsSubmit] = useState(false);
+  const [isCorrect, setIsCorrect] = useState("");
```

Comment:
Ideally, you only need `isCorrect`, and it should be a boolean rather than a string. A variable named in this manner signals that it should evaluate to true or false.